examples/petsc/bp1.c

```diff
@@ -78,18 +97,19 @@ static int CreateRestriction(Ceed ceed, const CeedInt melem[3],
 }
 
 static int Setup(void *ctx, void *qdata, CeedInt Q,
-                const CeedScalar *const *in, CeedScalar *const *out) {
+                 const CeedScalar *const *in, CeedScalar *const *out) {
   CeedScalar *rho = qdata, *target = rho + Q;
   const CeedScalar (*x)[Q] = (const CeedScalar (*)[Q])in[0],
-    (*J)[3][Q] = (const CeedScalar (*)[3][Q])in[1],
-    *w = in[4];
+                             (*J)[3][Q] = (const CeedScalar (*)[3][Q])in[1],
+                                          *w = in[4];
```

Comment:
This could be adjusted simply with
```c
  const CeedScalar (*x)[Q] = (const CeedScalar (*)[Q])in[0];
  const CeedScalar (*J)[3][Q] = (const CeedScalar (*)[3][Q])in[1];
  const CeedScalar *w = in[4];
```

Reply:
I don't like the redundancy, but the alternatives appear to be worse. 19a816d9244939dc5d19775b84ed2c38f712f81a